<launch>
  <param name="robot_no" type="str" value="tb3_1" />

  <node pkg="navi_gazebo" type="globalplanner.py" name="globalplanner_robot1" output="screen">
  </node>

  <node pkg="navi_gazebo" type="purepursuit.py" name="purepursuit_robot1" output="screen">
  </node>

</launch>
